src/crystal/project/github_dependency.cr

```diff
@@ -6,7 +6,7 @@ module Crystal
     getter target_dir
 
     def initialize(repo, name = nil : String?, ssh = false, branch = nil : String?)
-      unless repo =~ /(.*)\/(.*)/
+      unless repo =~ /^([^\/]*)\/([^\/]*)$/
```

Comment:
Well, why touching it, `"bar/baz\n"` passes this still ;) Start of string is `\A`, end of string is `\z`.